src/blocks/modal.css

```diff
@@ -179,10 +181,124 @@
 
 }
 
```

Comment:
* The project starts with errors https://snipboard.io/xglsNb.jpg . It's about the `space` in the file name https://snipboard.io/r0pl93.jpg . Please, rename the file into `DeleteIcon.svg` and correct the path in `card.css`
* **Space character** is not allowed in path segments: it results in validation error and may lead to unexpected results in the future, such as availability issues. If renaming the file is not an option, at least encode the path to be a valid `URL`. More info here http://www.w3schools.com/TAGS/ref_urlencode.asp